{"name": "test_add_course_fail[case0]", "status": "passed", "attachments": [{"name": "log", "source": "3a7aa23e-ad5e-4c74-aeb3-56e879351f4b-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "case", "value": "{'title': '验证码错误', 'verificate_code': 'P69YYY', 'expected': '该加课码不存在或者已经失效'}"}], "start": 1604474255585, "stop": 1604474257903, "uuid": "35f800b9-047c-4327-98f9-10976579c72a", "historyId": "08f41c2e50fa3306047a98754e972cd3", "testCaseId": "160dedc24855c98b250adf59f7822f75", "fullName": "testcase.test_02add_course.TestAddCourse#test_add_course_fail", "labels": [{"name": "parentSuite", "value": "testcase"}, {"name": "suite", "value": "test_02add_course"}, {"name": "subSuite", "value": "TestAddCourse"}, {"name": "host", "value": "ZBMAC-C02WD0XQH.local"}, {"name": "thread", "value": "18542-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.test_02add_course"}]}